scripts/carrot/BenchmarkComparisonOrchestrated.wdl

```diff
@@ -0,0 +1,346 @@
+        }
+
+        call BenchmarkAndRocCurves.BenchmarkComparison as NISTSampleHeadToHead {
+            input:
+                evalVcf = NIST_evalVcf,
+                evalLabel = NIST_evalLabel,
+                evalVcfIndex = NIST_evalVcfIndex,
+                evalRuntimeSummaries = NIST_evalRuntimeSummaries,
+                evalMonitoringExample = NIST_evalMonitoringExample,
+
+                controlVcf = NIST_controlVcf,
+                controlLabel = NIST_controlLabel,
+                controlVcfIndex = NIST_controlVcfIndex,
+                controlRuntimeSummaries = NIST_controlRuntimeSummaries,
+                controlMonitoringExample = NIST_controlMonitoringExample,
+
+                truthVcf = NIST_truthVcf,
+                confidenceInterval = NIST_confidenceInterval,
+                truthLabel = NIST_truthLabel,
+                truthVcfIndex = NIST_truthVcfIndex,
+
+                reference = reference,
+                refIndex = refIndex,
+                refDict = refDict,
+                hapMap = hapMap,
+                referenceVersion = referenceVersion,
+                stratIntervals = stratIntervals,
+                stratLabels = stratLabels,
+
+                monitoring_log_process_script = monitoring_log_process_script
+        }
+
+
+        call BenchmarkAndRocCurves.BenchmarkComparison as EXOME1SampleHeadToHead {
+            input:
+                evalVcf = EXOME1_evalVcf,
+                evalLabel = EXOME1_evalLabel,
+                evalVcfIndex = EXOME1_evalVcfIndex,
+                evalRuntimeSummaries = EXOME1_evalRuntimeSummaries,
+                evalMonitoringExample = EXOME1_evalMonitoringExample,
+
+                controlVcf = EXOME1_controlVcf,
+                controlLabel = EXOME1_controlLabel,
+                controlVcfIndex = EXOME1_controlVcfIndex,
+                controlRuntimeSummaries = EXOME1_controlRuntimeSummaries,
+                controlMonitoringExample = EXOME1_controlMonitoringExample,
+
+                truthVcf = EXOME1_truthVcf,
+                confidenceInterval = EXOME1_confidenceInterval,
+                truthLabel = EXOME1_truthLabel,
+                truthVcfIndex = EXOME1_truthVcfIndex,
+
+                reference = reference,
+                refIndex = refIndex,
+                refDict = refDict,
+                hapMap = hapMap,
+                referenceVersion = referenceVersion,
+
+                monitoring_log_process_script = monitoring_log_process_script
+        }
+
+        call CreateHTMLReport {
+            input:
+                merged_chm_plots = CHMSampleHeadToHead.rocPlotsmerged,
+                merged_nist_plots = NISTSampleHeadToHead.rocPlotsmerged,
+                merged_exome1_plots = EXOME1SampleHeadToHead.rocPlotsmerged,
+        }
+
+        output {
+            File ROC_Plots_Reported = CreateHTMLReport.report
+
+            File? CHMevalsummary=CHMSampleHeadToHead.evalsummary
```

Comment:
I don't think I knew it was possible to have optional outputs in WDL.  Under what circumstances would this not have a value?  Or am I misinterpreting what the question mark means in this context.

If it's possible for this workflow to produce an outputs JSON with certain of the outputs missing, that might cause an issue in carrot.